modules/sts-mode-installing-manual-run-installer.adoc

```diff
@@ -5,7 +5,40 @@
 [id="sts-mode-installing-manual-run-installer_{context}"]
 = Running the installer
 
-* Run the {product-title} installer:
+.Prerequisites
+
+* Create the `install-config.yaml` file for your installation.
+
+.Procedure
+
+. From the installation directory that contains your `install-config.yaml` file, configure the  Cloud Credential Operator (CCO) to run in manual mode:
++
+[source,terminal]
+----
+$ echo “credentialsMode: Manual” >> install-config.yaml
+----
+
+. Create the required installation manifests:
++
+[source,terminal]
+----
+$ openshift-install create manifests
+----
+
+. Copy the generated manifests to the installation directory:
+// Confused by this step from the enablement slide. Isn't the previous step run from the right directory already (if we're in the installation directory that contains the config YAML)?
```

Comment:
What needs to happen is that we save the manifest files created by `ccoctl` to the side, then allow the installer to do its thing and create its own separate manifests. Then we can finally merge the contents of the two directories by adding only the `ccoctl` manifests files next to the installer generated manifests files.

Reply:
aaaah ok, so `Copy the generated manifests to the installation directory` is `Copy the manifests that ccoctl generated to the manifests directory that the installation program created`